tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -551,7 +578,16 @@ def _start_on_server_thread(self) -> None:
         _logger.info("Starting WebTransport over HTTP/3 server on %s:%s",
                      self.host, self.port)
 
-        configuration.load_cert_chain(self.cert_path, self.key_path)
+        if self.cert_mode == WebTransportCertificateGeneration.USEPREGENERATED:
+            configuration.load_cert_chain(self.cert_path, self.key_path)
+        else: # GENERATEDVALIDSERVERCERTIFICATEHASHCERT case
+            configuration.private_key =  serialization.load_pem_private_key(self.cert_hash_info["private_key"],
+                                                                            password=None
+                                                                            )
+            configuration.certificate = x509.load_pem_x509_certificate(self.cert_hash_info["certificate"])
+            configuration.certificate_chain = []
```

Comment:
These are [commented as](https://github.com/aiortc/aioquic/blob/7f27a0b8fd2c4ab755c5a7ba32b4d432dda305dd/src/aioquic/quic/configuration.py#L103) "for internal purposes, not guaranteed to be stable".

As such, we probably need to write these out to some temporary files and call `configuration.load_cert_chain` to avoid causing ourselves pain down the road.

Reply:
Yes, but if you look at the implementation, it is very unlikely that these will change. But maybe a PR to the module to have a setter for the two would be a good idea. (Regarding temporary files, I was just not aware of how to achieve this safely in the WPT framework.)